so-widgets-bundle.php

```diff
@@ -617,6 +617,7 @@ function get_widgets_list(){
 			'AuthorURI' => 'Author URI',
 			'WidgetURI' => 'Widget URI',
 			'VideoURI' => 'Video URI',
+			'Documentation' => 'Documentation',
 		);
 
 		$widgets = array();
```

Comment:
Not really related to this PR, but while this is fresh please add the following after the `get_file_data` call in the loop below to make the file headers translatable: 
```
// Need to explicitly call `translate` for file headers.
foreach ( array( 'Name', 'Description' ) as $field ) {
	$widget[ $field ] = translate( $widget[ $field ], 'so-widgets-bundle' );
}
```